src/composables/watchers/useWeb3Watchers.ts

```diff
@@ -97,4 +99,10 @@ export default function useWeb3Watchers() {
       handlePendingTransactions();
     }
   });
+
+  watch(account, () => {
+    if (BLOCKED_ADDRESSES.includes(account.value)) {
+      disconnectWallet();
```

Comment:
I think that disconnecting a wallet is an elegant solution.

Blocked users which were previously connected will be disconnected... and if they try to connect they will disconnect right away.